node/script/test_websocketFetcher/main.go

```diff
@@ -2,91 +2,51 @@ package main
 
 import (
 	"context"
-	"time"
+	"encoding/json"
+	"sync"
 
-	"bisonai.com/orakl/node/pkg/common/keys"
-	"bisonai.com/orakl/node/pkg/db"
-	"bisonai.com/orakl/node/pkg/websocketfetcher"
 	"bisonai.com/orakl/node/pkg/websocketfetcher/common"
+	"bisonai.com/orakl/node/pkg/websocketfetcher/providers/coinbase"
 	"github.com/rs/zerolog/log"
 )
 
 func main() {
-
+	var wg sync.WaitGroup
 	ctx := context.Background()
-	feeds := []common.Feed{
+	feed := []common.Feed{
 		{
 			ID:         1,
-			Name:       "binance-wss-BTC-USDT",
-			Definition: nil,
+			Name:       "coinbase-wss-BTC-USDT",
+			Definition: json.RawMessage(`{"type": "wss", "provider": "coinbase", "base": "btc", "quote": "usdt"}`),
 			ConfigID:   1,
 		},
 		{
 			ID:         2,
 			Name:       "coinbase-wss-ETH-USDT",
-			Definition: nil,
+			Definition: json.RawMessage(`{"type": "wss", "provider": "coinbase", "base": "eth", "quote": "usdt"}`),
 			ConfigID:   2,
 		},
-		{
-			ID:         3,
-			Name:       "coinone-wss-BTC-KRW",
-			Definition: nil,
-			ConfigID:   3,
-		},
-		{
-			ID:         4,
-			Name:       "korbit-wss-BORA-KRW",
-			Definition: nil,
-			ConfigID:   4,
-		},
+		// {
```

Comment:
@nick-bisonai Are we planning to bring BORA and MBX back?

Reply:
this is test script I used while development, it can be removed